src/components/Newsletter/Newsletter.js

```diff
@@ -35,50 +42,131 @@ class Newsletter extends React.Component {
       });
   }
 
+  handleSelectChange = selectedOption => {
+    this.alertsSelectionArray = selectedOption;
+    this.setState({
+      alertSignUps: this.alertsSelectionArray
+        .map(item => item.full_fips_code)
+        .join(','),
+    });
+  };
+
   render() {
     const { stateId, county } = this.props;
+
+    const stateDataset = US_STATE_DATASET.state_dataset.map(state => {
+      return {
+        ...state,
+        full_fips_code: state.state_fips_code + '000',
+      };
+    });
+    const countyDataset = [];
+
+    each(US_STATE_DATASET.state_county_map_dataset, (value, key) => {
+      countyDataset.push(...value.county_dataset);
+    });
+
+    const autocomplete_options = stateDataset.concat(countyDataset);
```

Comment:
As I mentioned in chat, the overall site search gives me much better results if I try to search for "King County".  It filters out irrelevant stuff and sorts by population (or something) so my result is at the top.  I think we should try to do the same thing here.

![image](https://user-images.githubusercontent.com/206364/84073924-8e84c880-a986-11ea-92e0-209875c71e53.png)

![image](https://user-images.githubusercontent.com/206364/84073937-93497c80-a986-11ea-8024-6d801cabf589.png)

Reply:
I switched this to use a unique list (so solves some of the issue-- I guess the thing is that to use this in autocomplete + chip + other stuff it's just a lil easier to leave it like this now)